trivy.yaml

```diff
@@ -0,0 +1,29 @@
+timeout: 10m
+format: json
+dependency-tree: true
+list-all-pkgs: true
+exit-code: 1
+output: result.json
+# All severity levels
+severity:
+  - UNKNOWN
+  - LOW
+  - MEDIUM
+  - HIGH
+  - CRITICAL
+scan:
+  skip-dirs:
+    - .github/
+    - assets/
+    - core/.terraform/
+    - shared/.terraform/
+
+  scanners:
+    - vuln
+    - secret
+
+vulnerability:
+  type:
+    - os
+    - library
+  ignore-unfixed: true
```

Comment:
Small new line here 
```suggestion
  ignore-unfixed: true

```

Reply:
Updated in [ee0cb2a](https://github.com/nvminhtue/tyson_devops_ic/pull/19/commits/ee0cb2a91ff12ae5b534b4481c62a39d70ab2b45) 👍